exercises/04-integration/after/Divergent.Customers.ITOps/Customer.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Divergent.Customers.ITOps
+{
+    internal class Customer
```

Comment:
In this case the EF Context used here could directly map to `CustomerInfo` and then we wouldn't even need this type.

Reply:
@danielmarbach that sounds good, but I know next to nothing about EF. How can that be done?